src/tests/ftest/daos_test/suite.single_host.yaml

```diff
@@ -0,0 +1,159 @@
+# change host names to your reserved nodes, the
```

Comment:
Where is `src/tests/ftest/daos_test/suite.single_host.yaml` being used?

Reply:
It was (is, could be in the future) used in the Test RPMs stages that right now that duplicate some of the cluster setup and test that is done by Ftest.  There are still some issues to be worked out with that but I thought it was worth leaving the YAML file there for future work in that area.

Reply:
The ultimate problem is that running ftest requires a minimum of two nodes and the Test RPMs test only allocates one.  We don't have VM clusters of less than 9 nodes but more than 1.

Reply:
I think we could utilize `launch.py -e <path>` here. When you get around to testing with this config again, I could help you figure out the minimal config you need, to be applied on top of the "default" config.
Basically, if the default config is
```
foo: 1
bar: 2
```
And you use `launch.py -e <path>` where `path` is a config like this
```
bar: 3
```
The final test config used will be
```
foo: 1
bar: 3
```